axelar-chains-config/info/testnet.json

```diff
@@ -1981,6 +2007,11 @@
       "gasOptions": {
         "gasLimit": 50000000000
       },
+      "onchainGasEstimate": {
+        "chainName": "mantle",
+        "l1ChainName": "ethereum",
+        "gasEstimationType": 1
```

Comment:
change the estimation type

Reply:
We haven't released/deployed the arbitrum estimation yet.